arduino/walt/walt.ino

```diff
@@ -219,6 +219,8 @@ void process_command(char cmd) {
       init_vars();
       send_ack(CMD_RESET);
     } else if (cmd == CMD_GSHOCK) {
+      Serial.print(flip_case(cmd));
+      Serial.print(" ");
```

Comment:
The alternative option is to do send_trigger(gshock) this way there is no need for additional parsing of the number, but then it will be an odd command that replies with a trigger rather than a flipped case of itself.

In either case this will break the protocol and require people to update the Teensy side.
One way to make it a bit less painful is to add a command that replies with prtocol number which we will increment every time we break it. And if it's lower than Android side expects, the app will show a message instructing to update the teesny. With silent breakage we might get some frustrated users complaining about features breaking randomly.